functions-methods.js

```diff
@@ -33,4 +71,25 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(emailAdres) {
+    const indexOfAap = emailAdres.indexOf("@novi.nl");
+    if (emailAdres.endsWith("@novi.nl")) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Je check is ietswat te simpel als je kijkt wat er staat onder "Een e-mailadres is valide wanneer:".

Los van dat kun je direct de uitkomst van de expressie teruggeven:
```suggestion
    return emailAdres.endsWith("@novi.nl");
```